### DISCUSSION TITLE
OS Regression - Looking ideas on how to implement it

### DISCUSSION THREAD

**Discussion by thcoura** [BretFisher/ama#192] [Q&A]
Hi there, 

INTRODUCTION: I´m new with Docker and I´m excited with the many cool stuff which can be done. BTW I´m super happy with your course.

PROBLEM: In the company which I work, we are interested to do an experiment implementing an OS regression with Docker. We think that it can help us to identify "low hanging problems" before go into a real host with the claimed OSes supported.

GUESS: I believe that others here have being there with this idea or with some similar one.

CONTEXT: We have an almost self "contained" software, in case you don´t consider libc as a dependency :s, running an also self contained testcase.

QUESTION: I would like to hear from you and the community recommendations and possibly traps which I need to be aware of.

Thanks..

**Comment by BretFisher** [comment]
To me "OS" in Linux means "Kernel" which is not part of a container. Can you be more specific in what you want to test in a container?
